doc/modules/model_evaluation.rst

```diff
@@ -95,7 +95,6 @@ Scoring                           Function
 'neg_root_mean_squared_error'     :func:`metrics.mean_squared_error`
 'neg_mean_squared_log_error'      :func:`metrics.mean_squared_log_error`
 'neg_median_absolute_error'       :func:`metrics.median_absolute_error`
-'neg_mean_absolute_percentage_error' :func:`metrics.mean_absolute_percentage_error`
```

Comment:
don't remove it just reformat the table so it renders fine.

Reply:
changed as per my understanding. aligned with other lines.